src/examples/IK/exampleIK.cpp

```diff
@@ -0,0 +1,305 @@
+    {
+        std::cerr << "[error] Cannot read the iFeelSuit_vLink_Node_" + std::to_string(nodeNum)
+                         + " struct"
+                  << std::endl;
+        return false;
+    }
+    matioCpp::MultiDimensionalArray<double> data1
+        = vLink_node("data").asMultiDimensionalArray<double>();
+    if (!data1.isValid())
+    {
+        std::cerr << "[error] Cannot read the data field of the iFeelSuit_vLink_Node_"
+                         + std::to_string(nodeNum) + " struct"
+                  << std::endl;
+        return false;
+    }
+
+    size_t index0 = 0;
+    size_t index_w = 4;
+    size_t index_x = 5;
+    size_t index_y = 6;
+    size_t index_z = 7;
+
+    size_t angVel_x = 11;
+    size_t angVel_y = 12;
+    size_t angVel_z = 13;
+
+    iDynTree::Vector4 quat;
+
+    quat[1] = data1({index_x, index0, index_i});
+    quat[2] = data1({index_y, index0, index_i});
+    quat[3] = data1({index_z, index0, index_i});
+    quat[0] = data1({index_w, index0, index_i});
+
+    I_omeg_IMU[0] = data1({angVel_x, index0, index_i});
+    I_omeg_IMU[1] = data1({angVel_y, index0, index_i});
+    I_omeg_IMU[2] = data1({angVel_z, index0, index_i});
+
+    I_R_IMU = iDynTree::Rotation::RotationFromQuaternion(quat);
+
+    return true;
+}
+
+const std::vector<std::string> getJointsList()
+{
+    std::vector<std::string> nodesName;
+    nodesName.push_back("jT9T8_rotx");
+    nodesName.push_back("jT9T8_rotz");
+    nodesName.push_back("jRightShoulder_rotx");
+    nodesName.push_back("jRightShoulder_roty");
+    nodesName.push_back("jRightShoulder_rotz");
+    nodesName.push_back("jRightElbow_roty");
+    nodesName.push_back("jRightElbow_rotz");
+    nodesName.push_back("jLeftShoulder_rotx");
+    nodesName.push_back("jLeftShoulder_roty");
+    nodesName.push_back("jLeftShoulder_rotz");
+    nodesName.push_back("jLeftElbow_roty");
+    nodesName.push_back("jLeftElbow_rotz");
+    nodesName.push_back("jLeftHip_rotx");
+    nodesName.push_back("jLeftHip_roty");
+    nodesName.push_back("jLeftHip_rotz");
+    nodesName.push_back("jLeftKnee_roty");
+    nodesName.push_back("jLeftKnee_rotz");
+    nodesName.push_back("jLeftAnkle_rotx");
+    nodesName.push_back("jLeftAnkle_roty");
+    nodesName.push_back("jLeftAnkle_rotz");
+    nodesName.push_back("jLeftBallFoot_roty");
+    nodesName.push_back("jRightHip_rotx");
+    nodesName.push_back("jRightHip_roty");
+    nodesName.push_back("jRightHip_rotz");
+    nodesName.push_back("jRightKnee_roty");
+    nodesName.push_back("jRightKnee_rotz");
+    nodesName.push_back("jRightAnkle_rotx");
+    nodesName.push_back("jRightAnkle_roty");
+    nodesName.push_back("jRightAnkle_rotz");
+    nodesName.push_back("jRightBallFoot_roty");
+    nodesName.push_back("jL5S1_roty");
+
+    return nodesName;
+}
+
+void setTPoseThread()
+{
+    std::cout << "Enter 'calib' to run the T-pose calibration" << std::endl;
+    std::string input;
+    while (!stopThread)
+    {
+        std::cout << ">> ";
```

Comment:
I would remove all the `cout` and use `log()->`

Reply:
I substituted all the `cout` with the `log()->`.